### PR TITLE
Update code owners for CTF actions

### PR DESCRIPTION
**Pull request by lukaszcl**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,4 @@
-* @smartcontractkit/releng
\ No newline at end of file
+* @smartcontractkit/releng
+
+# Chainlink Testing Framework (CTF)
+/actions/ctf-* @smartcontractkit/test-tooling-team
```
Comment (lukaszcl) [inline CODEOWNERS:4]: @scheibinger do you think this will work for all ctf- folders in /actions?

Comment (erikburt) [reply CODEOWNERS:4]: I don't think so based on [these docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

```
# The `docs/*` pattern will match files like
# `docs/getting-started.md` but not further nested files like
# `docs/build-app/troubleshooting.md`.
docs/*  docs@example.com
```

Comment (scheibinger) [reply CODEOWNERS:4]: From what I recall, GH uses GLOB patterns, I checked with https://globster.xyz/
And based on that you pattern wouldn't work.

Maybe you could try 
```suggestion
/actions/ctf-*/** @smartcontractkit/test-tooling-team
```

**Review by scheibinger — CHANGES_REQUESTED** [review]
see the inline comment

File: CODEOWNERS (hunk 1 above)
Comment (lukaszcl) [reply CODEOWNERS]: thanks @scheibinger. I just applied your suggestion. Could you review again?

**Review by scheibinger — APPROVED** [review]

**Pull request merged**